neo/rawio/openephysrawio.py

```diff
@@ -255,14 +256,15 @@ def _parse_header(self):
         # Annotate some objects from coninuous files
         self._generate_minimal_annotations()
         bl_ann = self.raw_annotations['blocks'][0]
-        for seg_index in range(nb_segment):
-            seg_ann = bl_ann['segments'][seg_index]
+        for seg_index, oe_index in enumerate(oe_indices):
+            seg_ann = bl_ann['segments'][oe_index]
             if len(info['continuous']) > 0:
-                fullname = os.path.join(self.dirname, info['continuous'][seg_index][0])
+                fullname = os.path.join(self.dirname, info['continuous'][oe_index][0])
                 chan_info = read_file_header(fullname)
                 seg_ann['openephys_version'] = chan_info['version']
                 bl_ann['openephys_version'] = chan_info['version']
                 seg_ann['date_created'] = chan_info['date_created']
+                seg_ann['segment_index'] = oe_index + 1
```

Comment:
I would not use 'segment_index' because it is ambiguous but maybe
'origin_segment_index' or 'openephys_segment_index' or ...
Does openephyse is "one based" ?
Is the +1 relevent ?

Reply:
`openephys_segment_index` sounds better indeed. 
I do not really have an opinion on the +1. I added it because there is a subtraction happening when exploring the folder: see for example in the [docstring](https://github.com/NeuralEnsemble/python-neo/blob/62d8e3ccda1d379d5eb2eea8cc64504b74ba434e/neo/rawio/openephysrawio.py#L437) of `explore_folder`. I'll wait for your opinion before making another commit.

Reply:
OK. I forgot about this.
OE is quite inconsistent about one (seg) or zero (chans) based.
It make sense to keep thos +1 so.